user/timepix3/module/include/Timepix3Event2StdEventConverter.hh

```diff
@@ -15,6 +15,17 @@ namespace eudaq {
     static uint64_t m_syncTime;
     static uint64_t m_syncTime_prev;
     static size_t m_clearedHeader;
+
+    // configuration parameters:
+    static bool m_first_time;
+    static bool applyCalibration;
+    static std::string calibrateDetector;
```

Comment:
I can't find a reason why these have to be static members - you only use them locally in one function?

Reply:
True that, it's already changed by merging the latest caribou.